packages/core/src/config/config-resolvers.ts

```diff
@@ -139,7 +139,7 @@ export async function resolvePlugins(
         const absolutePluginPath = existsSync(maybeAbsolutePluginPath)
           ? maybeAbsolutePluginPath
           : // For plugins imported from packages specifically
-            require.resolve(plugin);
+            require.resolve(plugin, { paths: [configDir] });
```

Comment:
Is there one and only one config dir? What happens if there are multiple nested configs?

Reply:
The `node_modules` directory will always be in the root of the project. If the `configDir` is a nested directory, the plugin will still get resolved.